.pre-commit-config.yaml

```diff
@@ -43,6 +43,9 @@ repos:
     rev: 5.5.3
     hooks:
       - id: isort
+        args:
+          - --profile
+          - black
```

Comment:
This should not have been merged. This is defined in the project configuration file.